src/Assignment4/App.js

```diff
@@ -0,0 +1,10 @@
+import ReactDOM from "react-dom/client";
+
+import RootElement from "./Root";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <>
```

Comment:
The Fragment <></> is not needed here so you can remove it.

Reply:
removed.